kuma/static/styles/includes/_vars.scss

```diff
@@ -103,45 +103,50 @@ $grid-width-column-main : -calc-col-width(9);
 
 /*
 breakpoints & media queries
+- MDN should be coded "mobile first"
+- when coding features begin by coding the mobile presentation as default
+- use the min-width MQs to modify presentation for larger screens
 ====================================================================== */
 
-$max-width-default : 1400px;
-
-$small-desktop-ends : 1200px;
-$tablet-ends : 1024px;
-$mobile-ends : 768px;
-$small-mobile-ends : 480px;
-
-$small-desktop-starts : $tablet-ends + 1px;
-$tablet-starts : $mobile-ends + 1px;
-$mobile-starts : $small-mobile-ends + 1px;
-
-$media-query-small-desktop : 'all and (max-width: #{$small-desktop-ends})';
-$media-query-tablet : 'all and (max-width: #{$tablet-ends})';
-$media-query-mobile : 'all and (max-width: #{$mobile-ends})';
-$media-query-small-mobile : 'all and (max-width : #{$small-mobile-ends})';
-
-$media-query-between-small-desktop-and-max-width : 'all and (min-width: #{$small-desktop-starts}) and (max-width: #{$max-width-default})';
-
-$media-query-large-desktop-and-up : 'all and (min-width: #{$small-desktop-ends})';
-$media-query-small-desktop-and-up : 'all and (min-width: #{$small-desktop-starts})';
-$media-query-tablet-and-up : 'all and (min-width: #{$tablet-starts})';
-$media-query-mobile-and-up : 'all and (min-width: #{$mobile-starts})';
-
-$media-query-small-desktop-only : 'all and (min-width: #{$small-desktop-starts})';
-$media-query-tablet-only : 'all and (min-width: #{$tablet-starts}) and (max-width: #{$tablet-ends})';
-$media-query-mobile-only : 'all and (min-width: #{$mobile-starts}) and (max-width: #{$mobile-ends})';
-$media-query-small-mobile-only : $media-query-small-mobile;
-
-$navigation-break-ends : 1100px;
-$navigation-block-ends : 850px;
-
-$navigation-break-starts : $navigation-block-ends + 1px;
-
-$media-query-navigation-break : 'all and (max-width: #{$navigation-break-ends})'; /* Moves nav items all the way to the left, moves logo above */
-$media-query-navigation-block : 'all and (max-width: #{$navigation-block-ends})'; /* Moves nav menu items to block, its own line */
-
-$media-query-break-and-up : 'all and (min-width: #{$navigation-break-starts})';
+/* break points */
+$max-width-default : (1400px / 16px) * 1em;
```

Comment:
Question: Should we put a note here to explain that what this is doing is converting pixel widths to it's equivalent em width? Made immediate sense to me but, maybe would not be clear to everyone. Teachable moment?